app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -99,7 +99,7 @@ class AdministratorControlsActivityTest {
   @Test
   // TODO(#973): Fix AdministratorControlsActivityTest
   @Ignore
-  fun testAdministratorControlsActivity_withAdminProfile_openAdministratorControlsActivityFromNavigationDrawer_onBackPressed_showsHomeActivity() { // ktlint-disable max-line-length
+  fun testAdminControlsActivity_withAdminProfile_openAdminCtrlsActivityFromNavDrawer_HomeOnBack() {
```

Comment:
Suggestion:
`testAdminCtrlActivity_withAdminProfile_openAdminControlsActivityFromNavDrawer_onBackPressed_showsHomeActivity`

If the above does not work then

`testAdminCtrl_withAdminProfile_openAdminControlsFromNavDrawer_onBackPressed_showsHome`